thoth/storages/graph/postgres.py

```diff
@@ -4611,12 +4619,12 @@ def _get_or_create_python_package_index(
         session: Session, index_url: str, only_if_enabled: bool = True
     ) -> Optional[PythonPackageIndex]:
         """Get or create Python package index entry with a check the given index is enabled."""
+        index_url = GraphDatabase.map_python_index_url(index_url)
```

Comment:
Nice, we can do this for all the functions in this `postgres.py` module that accept index_url.

Reply:
Added it in all functions having that parameter in `postgres.py`. Btw don't all of them ultimately depend on - `_get_or_create_python_package_index`?